spec/src/main/asciidoc/microprofile-lra-spec.adoc

```diff
@@ -323,10 +344,10 @@ this identifier is passed as a method parameter.
 The implementation MUST propagate the `active context` on outgoing JAX-RS
 requests.
 
-If an LRA is propagated to a resource that is not annotated with any of the
-annotations defined in this specification then the LRA will not be made
-available to the invoked method. However, any outgoing JAX-RS invocations
-made during the execution of the method will still carry the original context.
+If an LRA is propagated to a resource that is not annotated with `@LRA` then the implementation behaves
+as if there is an annotation present on the resource with the `type()` defined by the configuration parameter
+`mp.lra.LRA.value`. From a functional point of view, there is no difference notable except for the different
+default for `end()` which is false when there is no `@LRA` defined.
```

Comment:
If we go with a boolean valued property this text needs to be reworded.

Reply:
@rdebusscher this still references the old configuration parameter mp.lra.LRA.value.

Reply:
+1